diesel_migrations/migrations_macros/src/embed_migrations.rs

```diff
@@ -47,7 +47,7 @@ fn migration_literal_from_path(path: &Path) -> proc_macro2::TokenStream {
     if version_from_string(&name).is_none() {
```

Comment:
Should we handle the migrations without down migration for `embed_migrations` as well? I mean we could just convert accept an `Option` as argument there + error at runtime similarly to what you've implemented for file based migrations.